05_06_19_DTR.md

```diff
@@ -0,0 +1,56 @@
+#Links
```

Comment:
Thanks for typing this. Would it be okay with you if we added this file to a Google Doc instead of inside of the project?